src/routers/blog.js

```diff
@@ -69,10 +72,19 @@ router.get('/feed.xml', async (request, response) => {
 
 router.get('/:slug', async (request, response, next) => {
 
-	const reqLocale = request.locale;
-	const locale = util.getLocale(reqLocale.region, reqLocale.language);
+	const renderData = 	{
+		layout: 'blog-opengraph',
+		locale: util.getLocale(request.locale.region, request.locale.language),
+		localeString: request.locale.toString(),
+		postList,
+	};
 
-	const localeString = reqLocale.toString();
+	renderData.isLoggedIn = request.cookies.access_token && request.cookies.refresh_token && request.cookies.ph;
+
+	if (renderData.isLoggedIn) {
+		const account = await util.getAccount(request, response);
+		renderData.account = account;
+	}
```

Comment:
this can be made a middleware tbh, its repeated like 50 times

Reply:
its going to be made into middleware. After all these features are merged I'm doing a big rewrite of all this auth handling